stdlib/std.jsonnet

```diff
@@ -660,7 +660,7 @@ limitations under the License.
           local tmp = if code.fw == '*' then {
             j: j + 1,
             fw: if j >= std.length(arr) then
-              error 'Not enough values to format: ' + std.length(arr)
+              error ('Not enough values to format: ' + std.length(arr) + ', expected ' + j)
```

Comment:
we don't actually know how many are expected because we haven't gotten to the end of `codes` yet.  So it could say `"expected at least " + j` or something like that (the same in the other cases).

Reply:
Alternatively could use something like `std.length(std.filter(codes, function(e)std.isObject(e)))`

Reply:
Oh you also have to count the ones with `*` twice, so that doesn't work.

Reply:
I guess could map it to either 0 1 or 2 and then use std.foldl with a +

Reply:
`expected at least` still makes sense and is helpful so I changed it to that for now and fixed the other variable issues as well. :) Thanks for reviewing.